celer/group_fast.pyx

```diff
@@ -89,7 +89,7 @@ cpdef floating dnorm_grp(
 
     else:  # scaling only with features in C
         for g_idx in range(ws_size):
-            if weights[g] == INFINITY:
+            if weights[g_idx] == INFINITY:
                 continue
 
             g = C[g_idx]
```

Comment:
are you sure that it's not g (defined L95, so def should be moved above) which should be used here? weights has size n_groups, not ws_size

Reply:
Yes you are right, 
We can insert line 95 right after Line 91 in we keep the rest as it is